server/config/db/pouchdb.js

```diff
@@ -14,19 +18,52 @@ function pouchDbConfig (state, callback) {
     throws: false
   }) || {}
   var secret = store.couch_httpd_auth_secret
+  var adminPassword = get(store, 'admins.admin')
 
   if (!secret) {
     secret = randomstring.generate({
       charset: 'hex'
     })
-    jsonfile.writeFileSync(storePath, Object.assign(store, {
-      couch_httpd_auth_secret: secret
-    }), {spaces: 2})
+
+    if (adminPassword) {
+      jsonfile.writeFileSync(storePath, Object.assign(store, {
+        couch_httpd_auth_secret: secret
+      }), {spaces: 2})
+    }
   }
 
   state.config.db.secret = secret
-  state.config.db.admins = {}
+  state.config.db.admins = store.admins
   state.config.db.authenticationDb = '_users'
 
-  callback(null, state.config)
+  if (adminPassword) {
+    return callback(null, state.config)
+  }
+
+  var admins = new Admins({
+    secret: secret
+  })
+
+  admins.set('admin', 'secret')
```

Comment:
here is where we currently set the admin password to "secret", which of course is only a short temporary solution (without it admins cannot sign in at all). I’ll create a follow up issue to set the password per CLI